src/api/language.js

```diff
@@ -24,6 +25,20 @@ const loadMessages = (lang) => {
     .then((messages) => Object.assign.apply({}, messages))
 }
 
+const loadAllLanguagesMessages = () => {
+  const messages = {}
+  allLanguages.forEach(language => {
+    try {
+      messages[language.lang] = require(`../../i18n/${language.lang}.json`)
```

Comment:
Can you simplify the path since you have the webpack alias?

Reply:
The following is not working for me:

```
messages[language.lang] = require(`i18n/${language.lang}.json`)
```

Seems to be an open issue others are seeing: https://github.com/tleunen/babel-plugin-module-resolver/issues/291

Do you see the same behavior? Note this was working with `import` like we did with commit 3d2cb9e33f8747bb5d37d66c6580011a35bc3c03

If you're seeing the same behavior, I can go ahead and [add a comment](https://github.com/tleunen/babel-plugin-module-resolver/issues/291) to signal urgency